update-arcdps.ps1

```diff
@@ -108,17 +113,34 @@ if ($run_update -eq $false) {
         Move-Item $templates_path $templates_backup
     }
 
+    # Also backup extras
+    if (X-Test-Path $extras_path) {
+        if (X-Test-Path $extras_backup) {
+            Remove-Item $extras_backup
+        }
+        Move-Item $extras_path $extras_backup
+    }
+
     # Remove the copy in bin64 as well
     if (X-Test-Path $arc_bin_path) {
         Remove-Item $arc_bin_path
     }
+    if (X-Test-Path $templates_bin_path) {
+        Remove-Item $templates_bin_path
+    }
+    if (X-Test-Path $extras_bin_path) {
+        Remove-Item $extras_bin_path
+    }
```

Comment:
No need to fix in this patch, but it might be worth switching this to an object to reduce duplicate code. something to leave as a left over project.